docs/policy/comanage-instructions-user.md

```diff
@@ -0,0 +1,54 @@
+Registering for the OSG COManage
+================================
+
+OSG is using a new identity management system called COManage.
+This system will be used for managing access to OSG staff-internal monitoring webpages at UNL
+and managing access for OASIS logins.
+
+Follow the instructions below to register with COManage and join the OSG group.
+
+!!! note
+    This page is for users who want to register with COManage.
+    If you are a COManage Admin and want to approve a user registration,
+    go to the [Approving COManage Registrations](/policy/comanage-instructions-admin) page instead.
+
+
+1.  Go to <https://registry.cilogon.org/registry/co_petitions/start/coef:64>
+
+1.  Search for your institution and select it.
+
+1.  Log in with your institutional credentials.
+    You should reach the "OSG Registration: Self Signup With Approval" page.
+
+1.  Click "BEGIN".
+
+1.  Enter your name and email address in the form fields;
+    defaults might be filled in from what your institution provided,
+    but you are permitted to change them.
```

Comment:
Add a note about it being OK to use a different email address here than used to login?